test/e2e/util.go

```diff
@@ -170,6 +170,50 @@ func waitForPodSuccess(c *client.Client, podName string, contName string) error
 	return waitForPodSuccessInNamespace(c, podName, contName, api.NamespaceDefault)
 }
 
+// Context for checking pods responses by issuing GETs to them and verifying if the answer with pod name.
+type podResponseChecker struct {
+	c              *client.Client
+	ns             string
```

Comment:
"ns" is a commonly used abbreviation for namespace in kubernetes, and, AFAK, golang standard in general encourages short variable names.